chain/db/src/db.rs

```diff
@@ -509,6 +511,17 @@ impl<T: AsyncKVStorage<Key = Box<[u8]>, Value = Box<[u8]>> + Clone + Send + Sync
         Ok(agg_tickets)
     }
 
+    async fn get_acknowledged_ticket(
+        &self,
+        channel_id: &Hash,
+        epoch: u32,
```

Comment:
What sense does it make to have a get acked ticket from a previous epoch? It cannot be redeemed, right? Should it rather not be discarded automatically by the DB on epoch update?

Reply:
it's just a general getter function. And yes, you're right. It doesn't make sense to get tickets from previous channel instantiations